src/main/java/org/myrobotlab/service/InMoov.java

```diff
@@ -996,7 +1066,7 @@ public InMoovHead startHead(String port, Integer headYPin, Integer headXPin, Int
   public InMoovHead startHead(String port, String type, Integer headYPin, Integer headXPin, Integer eyeXPin, Integer eyeYPin, Integer jawPin, Integer rollNeckPin)
       throws Exception {
     // log.warn(InMoov.buildDNA(myKey, serviceClass))
-    speakBlocking(String.format("starting head on %s", port));
+    speakBlocking(languagePack.get("STARTINGHEAD")+" "+port);
```

Comment:
I'm not sure I changed this ( just the text ) , also I think speablocking instead of speak is important to avoid concurent speech events, or speaking for events that are already done because fast launched... ( also mute is possible at startup )